full/src/test/java/apoc/custom/CypherProceduresTest.java

```diff
@@ -839,6 +847,23 @@ public void testIssue2032() {
         assertProcedureFails(String.format(SIGNATURE_SYNTAX_ERROR, procedureSignature), 
                 "call apoc.custom.declareProcedure('" + procedureSignature + "','RETURN $first + $s AS answer')");
     }
+
+    @Test
+    public void testIssue3349() {
+        String procedure = "CALL apoc.custom.declareProcedure(\n" +
```

Comment:
The issue asks for a void return type from what I can tell, this looks like it is testing not having any parameters to pass in, can a test be added for testing a void return? Or will that be separate?

Reply:
@gem-neo4j thanks for the question, the issue has an example with the following signature:
`setrole(ssorole::STRING) :: (row::MAP)`, that triggers an error, then the user said that he also tested with `VOID` return type. So as the issue is not related to the return type but to the fact (as discussed in the trello card) that was not meant to manage `DBMS` and `SCHEMA` query types, I used an example as much similar as possible to the one shared in the GH issue (https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/3349).

Btw if you want I can add a test also with the return type.

Do you also want a test with a custom procedure with parameters? With `VOID` or another return type?

Reply:
It would be nice to have, considering the title of this fix specifies it is for void return type :)

Reply:
@gem-neo4j done!